packages/integration-tests/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Maybe there will be some collision with #4794

Reply:
file removed